bluemira/neutronics/make_materials.py

```diff
@@ -0,0 +1,307 @@
+"""Create the material sets for each type of reactor."""
+from __future__ import annotations
+
+import dataclasses
+import os
+from dataclasses import dataclass
+from enum import Enum, auto
+from typing import Union
+
+import openmc
+from openmc import Material
+
+import bluemira.neutronics.materials_definition as md
+
+
+def duplicate_mat_as(mat_to_clone, new_id, new_name) -> Material:
+    """Clones and renames an OpenMC material"""
+    new_mat = mat_to_clone.clone()
+    new_mat.id = new_id
+    new_mat.name = new_name
+
+    return new_mat
+
+
```

Comment:
Can we rationalise the whole materials stuff in the materials module with this? @je-cook @OceanNuclear 

It might be time to decouple from neutronics material maker if we can just do everything in OpenMC... Not sure how much work is implied here, I haven't looked at out materials stuff in years.

Reply:
We will do this but I'd like to get this in to the feature branch first